data-model/pom.xml

```diff
@@ -87,6 +87,14 @@
                     </args>
                 </configuration>
             </plugin>
+            <plugin>
```

Comment:
Is this plugin needed in `data-model`? Try rolling it back if the whole thing builds.

Reply:
* Was not rolled back (maybe it's needed, then add the comment) 😉 
* Please don't close a comment as `Resolved` when somebody else started it unless it's a _Suggestion_ you accepted